tests/foreman/ui/test_activationkey.py

```diff
@@ -111,19 +96,19 @@ def test_positive_end_to_end_register(session):
     org = entities.Organization().create()
     lce = entities.LifecycleEnvironment(organization=org).create()
     repos_collection = RepositoryCollection(
-        distro=DISTRO_RHEL7, repositories=[SatelliteToolsRepository()]
+        distro=constants.DISTRO_RHEL7, repositories=[SatelliteToolsRepository()]
     )
     repos_collection.setup_content(org.id, lce.id, upload_manifest=True)
     ak_name = repos_collection.setup_content_data['activation_key']['name']
-    with VirtualMachine(distro=DISTRO_RHEL7) as vm:
-        repos_collection.setup_virtual_machine(vm)
-        with session:
-            session.organization.select(org.name)
-            chost = session.contenthost.read(vm.hostname, widget_names='details')
-            assert chost['details']['registered_by'] == f'Activation Key {ak_name}'
-            ak_values = session.activationkey.read(ak_name, widget_names='content_hosts')
-            assert len(ak_values['content_hosts']['table']) == 1
-            assert ak_values['content_hosts']['table'][0]['Name'] == vm.hostname
+
+    repos_collection.setup_virtual_machine(rhel7_contenthost)
```

Comment:
I'll be changing this again in #8665 as this method gets moved to `ContentHost` class - but for now, it's like this.

Reply:
I was thinking of a different method - this one from `robottelo.products.RepositoryCollection` stays.